DigitalLearningSolutions.Web/Views/TrackingSystem/CentreConfiguration/RegistrationPrompts/AddRegistrationPromptConfigureAnswers.cshtml

```diff
@@ -13,70 +13,50 @@
 }
 
 @section NavMenuItems {
-  <partial name="~/Views/TrackingSystem/Shared/_NavMenuItems.cshtml"/>
-}
-
-@if (errorHasOccurred) {
-  <div class="nhsuk-grid-row">
-    <div class="nhsuk-grid-column-full">
-      <vc:error-summary order-of-property-names="@new []{ nameof(EditRegistrationPromptViewModel.OptionsString), nameof(EditRegistrationPromptViewModel.Answer)}"/>
-    </div>
-  </div>
+  <partial name="~/Views/TrackingSystem/Shared/_NavMenuItems.cshtml" />
 }
 
 <div class="nhsuk-grid-row">
   <div class="nhsuk-grid-column-full">
+    @if (errorHasOccurred) {
+      <vc:error-summary order-of-property-names="@new []{ nameof(EditRegistrationPromptViewModel.OptionsString), nameof(EditRegistrationPromptViewModel.Answer)}" />
+    }
+
     <h1 class="nhsuk-heading-xl">Configure answers</h1>
-  </div>
-</div>
 
-<form class="nhsuk-u-margin-bottom-8" method="post" novalidate asp-action="AddRegistrationPromptConfigureAnswers">
-  <div class="hidden-submit">
-    <button name="action" class="nhsuk-button" value="@RegistrationPromptsController.AddPromptAction">Add</button>
-  </div>
+    <form class="nhsuk-u-margin-bottom-3" method="post" novalidate asp-action="AddRegistrationPromptConfigureAnswers">
+      <div class="hidden-submit">
+        <button name="action" class="nhsuk-button" value="@RegistrationPromptsController.AddPromptAction">Add</button>
+      </div>
 
-  <input type="hidden" asp-for="OptionsString"/>
+      <input type="hidden" asp-for="OptionsString" />
 
-  <div class="nhsuk-grid-row">
-    <div class="nhsuk-grid-column-full">
       @if (string.IsNullOrEmpty(Model.OptionsString)) {
-        <partial name="_NoConfiguredAnswers"/>
+        <partial name="_NoConfiguredAnswers" />
       } else {
-        <partial name="_RegistrationPromptAnswerTable" model="Model"/>
+        <partial name="_RegistrationPromptAnswerTable" model="Model" />
       }
-    </div>
-  </div>
-
-  <div class="nhsuk-grid-row divider">
-    <div class="nhsuk-grid-column-one-half">

```

Comment:
Does removing this `nhsuk-grid-column-one-half` make any difference to the appearance of the page (inc. on mobile)?

Reply:
Not significantly, I've change from a half-column with a full width textbox to a full width column with a half width textbox.